### PR TITLE
THREAT-420: Stratus AWS Logging Detections

### PR DESCRIPTION
**Pull request by ben-githubs**
### Background

Here's some rules for modifying cloudtrail logs. We can test the with Stratus.

### Changes

- added new rules

### Testing

- the new rules have tests!

### CODE REVIEW

File: rules/aws_cloudtrail_rules/aws_cloudtrail_short_lifecycle.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            "lifecycleRuleDurationDays": duration,
+        }
+    )
+
+    return context
+
+
+def is_cloudtrail_bucket(bucket_name: str) -> bool:
+    """Returns True if the bucket is known to contain CloudTrail logs."""
+    return bucket_name in CLOUDTRAIL_BUCKETS
```
[the remark this answers is not in the document]
Comment (ben-githubs) [reply rules/aws_cloudtrail_rules/aws_cloudtrail_short_lifecycle.py:81]: Normally, no, but in order to mock them for reliable unit tests, we need them to be functions :(

File: rules/aws_cloudtrail_rules/aws_cloudtrail_short_lifecycle.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
+from panther_aws_helpers import aws_cloudtrail_success, aws_rule_context, lookup_aws_account_name
+from panther_base_helpers import deep_get
+
+# Use this to record the names of your S3 buckets that have cloudtrail logs
+#   If a bucket name isn't mentioned here, we still make a best guess as to whether or not it
+#   contains CloudTrail data, but the confidence rating will be lower, and so will the severity
+CLOUDTRAIL_BUCKETS = "example_cloudtrail_bucket_name"
```
[the remark this answers is not in the document]
Comment (ben-githubs) [reply rules/aws_cloudtrail_rules/aws_cloudtrail_short_lifecycle.py:7]: 100%, not sure how I missed that! Will get it updated

File: rules/aws_cloudtrail_rules/aws_cloudtrail_event_selectors_disabled.yml [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+  - Security Control
+  - Defense Evasion:Impair Defenses
+Reports:
+  CIS:
+    - 3.5
+  MITRE ATT&CK:
+    - TA0005:T1562
+Severity: Medium
+Description: >
+  A CloudTrail Trail was modified.
```
[the remark this answers is not in the document]
Comment (ben-githubs) [reply rules/aws_cloudtrail_rules/aws_cloudtrail_event_selectors_disabled.yml:19]: Yep, I agree. Will make that change today!

File: rules/aws_cloudtrail_rules/aws_cloudtrail_short_lifecycle.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (arielkr256) [reply rules/aws_cloudtrail_rules/aws_cloudtrail_short_lifecycle.py:81]: Oh right, carry on

**Review by arielkr256 — APPROVED** [review]
LGTM!

**Pull request merged**